test/performance/odp_dma_perf_run.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+#
+# Copyright (c) 2022, Nokia
+# All rights reserved.
+#
+# SPDX-License-Identifier:	BSD-3-Clause
+
+TEST_DIR="${TEST_DIR:-$(dirname $0)}"
+BIN_NAME=odp_dma_perf
+ROUNDS=1000000
+
+check_result()
+{
+	if [ $1 -ne 0 ]; then
+		echo "Test FAILED, exiting"
+		exit 1
+	fi
+}
+
+echo "odp_dma_perf: synchronous transfer 1"
+echo "===================================="
+
+${TEST_DIR}/${BIN_NAME}${EXEEXT} -t 0 -g 0 -i 6 -s 10240 -T 0 -r $ROUNDS
+
+check_result $?
+
+echo "odp_dma_perf: synchronous transfer 2"
+echo "===================================="
+
+${TEST_DIR}/${BIN_NAME}${EXEEXT} -t 0 -g 1 -i 6 -s 10240 -T 0 -r $ROUNDS
```

Comment:
Some pool implementations may not be able to allocate this large packets (6 * 10240), so it could perhaps be safer to use the default 1024B segment size. The `odp_dma_capability_t.max_seg_len` could also be a limiting factor.

Reply:
Fixed in v2.